packages/x-components/src/x-modules/history-queries/components/clear-history-queries.vue

```diff
@@ -5,6 +5,7 @@
     :disabled="isHistoryQueriesEmpty"
     :events="clearHistoryQueriesEvents"
     data-test="clear-history-queries"
+    aria-label="Clear all"
```

Comment:
In case a setup gives a meaningful content to the slot below, which element has a preference? The `aria-label` or that content? If it is the `aria-label` it may force us to also override this "Clear all" with another value for the `aria-label` more related to the setup itself 🤔

Reply:
In case you need to substitute it, you can use `aria-labelledby`, placing it together with the id:
```
      <div>
        <h1>ClearHistoryQueries and RemoveHistoryQuery</h1>
        <SearchInput />
        <ClearHistoryQueries aria-labelledby="here" id="here">Not reading this since there is an aria-label</ClearHistoryQueries>
        <HistoryQueries :maxItemsToRender="3" />
      </div>
```
This way we ensure the button will be accessible by default but we can also change the content if needed.